runtime/compiler/optimizer/FearPointAnalysis.cpp

```diff
@@ -36,7 +36,7 @@
 
 bool TR_FearPointAnalysis::virtualGuardsKillFear()
    {
-   static bool kill = (feGetEnv("TR_FPAnalaysisGuardsDoNotKillFear") == NULL);
+   static bool kill = (feGetEnv("TR_FPAnalysisGuardsDoNotKillFear") == NULL);
```

Comment:
@andrewcraik should this be "fixed". Who should we ask?

Reply:
I would leave that alone for now. Just to make sure we don't break anything.